namespace-to-cjs.rb

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env ruby
+
+require 'find'
+require 'fileutils'
+require 'pry'
+
+
+
+# Constants
+
+CLASS_EXPRESSION = /^class\s(?:@)?([A-Z][\w\.]*)/
+DEPENDENCY_EXPRESSION = /^(?!(\s\*|#)).*(?:new|extends)\s(Brainstem[\w\.]*)/
+
+REQUIRE_TEMPLATE = "%{short_name} = require('%{path}')"
+
+# Globals
+
+$class_map = {}
+$files = []
+
+
+
+# Helpers
+
+def enumerate_files
+  Find.find('vendor/assets/javascripts/') do |path|
```

Comment:
Might be interesting to consider how we could pass this path in and the namespace (Brainstem) above.